src/components/SoqlListView.component

```diff
@@ -29,148 +29,195 @@
+                                <td class="slds-text-body--small slds-cell-wrap">
+                                    <apex:outputText value="{!item.sobj[f2]}" />
+                                </td>
+                            </apex:outputText>
+                        </apex:repeat>
+                    </tr>
+                </apex:repeat>
+           </tbody>
+        </table>
 
-				<apex:repeat value="{!pageController.strFields}" var="f" > 
-					<apex:column value="{!item.sobj[f]}" rendered="{!f != pageController.strNameField}" >
-						<apex:facet name="header"> 
-							<apex:outputPanel > <!-- needed only one element under header -->
-								<apex:outputText value="{!$ObjectType.Contact.Fields[f].Label}" rendered="{!$ObjectType.Contact.Fields[f].Type == 'multipicklist'}" />
-								<apex:commandLink action="{!SortItem}" rerender="panelData" status="statusLoad" value="{!$ObjectType.Contact.Fields[f].Label}" 
-									rendered="{!$ObjectType.Contact.Fields[f].Type != 'multipicklist'}" >
-									<apex:param assignTo="{!SortItemField}" name="SortItemField" value="{!f}" />
-									<apex:outputText style="font-size:1.0em;font-weight:bold;font-family:symbol" 
-										value="{!IF(SortItemField=f, IF(SortItemAsc, ' ↑' , ' ↓'), '')}" escape="false" />
-								</apex:commandLink>
-							</apex:outputPanel>
-						</apex:facet>
-					</apex:column> 
-	  			</apex:repeat>         
-	            	            
-	        </apex:pageBlockTable> 
-			<br/>
-			<div class="PagerUI">
-				<apex:panelGrid columns="12" > 
-					<apex:outputText value="" />
-					<apex:commandLink action="{!setCon.first}" rerender="panelData" status="statusLoad" >{!$Label.labelListViewFirst}</apex:commandlink>
-					<apex:outputText value=" | " />
-					<apex:commandLink action="{!setCon.previous}" rerender="panelData" status="statusLoad" rendered="{!setCon.HasPrevious}">{!$Label.labelListViewPrevious}</apex:commandlink> 
-					<apex:outputText rendered="{!setCon.HasPrevious}" value=" | " />
-					<apex:commandLink action="{!setCon.next}" rerender="panelData" status="statusLoad" rendered="{!setCon.HasNext}">{!$Label.labelListViewNext}</apex:commandlink>
-					<apex:outputText rendered="{!setCon.HasNext}" value=" | " />
-					<apex:commandLink action="{!setCon.last}" rerender="panelData" status="statusLoad" >{!$Label.labelListViewLast}</apex:commandlink>
-					<apex:outputText value="" />
-					<apex:outputText value="{!$Label.labelListViewPageInfo}" escape="false" >
-						<apex:param value="{!setCon.PageNumber}" />
-						<apex:param value="{!NumberOfPages}" />
-						<apex:param value="{!NumberOfItems}" />
-					</apex:outputText> 
-					<apex:outputText value="" />
-					<apex:commandLink action="{!showMoreRecordsPerPage}" rerender="panelData" status="statusLoad" >{!$Label.labelShowMore}</apex:commandlink>				   	
-				</apex:panelGrid> 
-			</div>		
-	    </apex:pageBlock>       	       	
-	</apex:outputPanel>
+        <div class="PagerUI">
```

Comment:
the bottom pager ui is kinda messed up.  it spreads out across the page.  the show more link is beyond the right of what this picture shows.
![image](https://cloud.githubusercontent.com/assets/908119/21283105/c1533472-c3b1-11e6-9581-df5c752e8ade.png)